Makefile

```diff
@@ -6,10 +6,22 @@
 #
 # SPDX-License-Identifier: ISC
 
+.PHONY: build
+build:
+	python setup.py build
+
+.PHONY: install
+install:
+	python setup.py install
+
 PYTHON_FORMAT ?= yapf
 format:
-	$(IN_ENV) find . -name \*.py $(FORMAT_EXCLUDE) -print0 | xargs -0 -P $$(nproc) yapf -p -i
+	$(IN_ENV) git ls-files | grep -ve '^third_party\|^\.' | grep -e '.py$$' | xargs -P $$(nproc) yapf -p -i
```

Comment:
Why did this change?

Reply:
Because it was trying to format `third_party` and the `.tox` env, which takes a long time and then of course fails.

Reply:
It also didn't handle generated files, such as `fasm/tags.py`. This ensures that formatting is only checked for checked in files.